src/components/Error/ErrorMessage.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+import CloseButton from '../Button/CloseButton';
+
+interface Props {
+  errorMessage: string;
+  onClose?: () => void;
+}
+
+const ErrorMessage: React.FC<Props> = ({ errorMessage, onClose }) => {
+  return (
+    <ErrorBanner>
+      <Error>{errorMessage}</Error>
+      <CloseButton onClick={onClose} top={0} right={0} />
+    </ErrorBanner>
+  );
+};
+
+const ErrorBanner = styled.div`
+  position: relative;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
Generally position `absolute` is used here in case you wanna centre an element within a container, any specific reason for using `relative`?